kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiDefaultStream.java

```diff
@@ -110,7 +116,9 @@ public void appendRows(TableName tableName, List<Object[]> rows, String streamNa
         JSONArray jsonArr = new JSONArray();
         JsonStreamWriter writer = getDefaultStream(tableName.toString());
 
-        rows.forEach(item -> jsonArr.put(item[1]));
+        for (Object[] item : rows) {
+            jsonArr.put(item[1]);
+        }
```

Comment:
What is the reason behind making this change?

Reply:
lambda functions expect effectively final variables.